tests/cases/data/source/MongoDbTest.php

```diff
@@ -911,206 +710,11 @@ public function testGeoQueries() {
-			]
-		]);
-
-		$trace = $db->server->trace;
-		$this->assertEqual('__call', $trace[1][0]);
-		$this->assertEqual('setReadPreference', $trace[1][1][0]);
-		$this->assertEqual('__get', $trace[2][0]);
-		$this->assertEqual('test', $trace[2][1][0]);
-	}
-
-	public function testDefaultSafeOptions() {
-		$this->_db = new MongoDb($this->_testConfig + ['w' => 1, 'wTimeoutMS' => 10000]);
-		$this->_db->server = new MockMongoConnection();
-		$this->_db->connection = new MockCallable();
-		$this->_db->connection->custom = new MockCallable();
-
-		$query = new Query(['type' => 'read', 'source' => 'custom']);
-		$this->_db->create($query);
-		$result = $this->_db->connection->custom->call;
-		$expected = [null, ['w' => 1, 'wTimeoutMS' => 10000, 'fsync' => false]];
-		$this->assertEqual('insert', $result['method']);
-		$this->assertEqual($expected, $result['params']);
-
-		$query = new Query(['type' => 'read', 'source' => 'custom', 'data' => ['something']]);
-		$this->_db->update($query);
-		$result = $this->_db->connection->custom->call;
-		$expected = ['upsert' => false, 'multiple' => true, 'w' => 1, 'wTimeoutMS' => 10000, 'fsync' => false];
-		$this->assertEqual('update', $result['method']);
-		$this->assertEqual($expected, $result['params'][2]);
-
-		$query = new Query(['type' => 'read', 'source' => 'custom']);
-		$this->_db->delete($query);
-		$result = $this->_db->connection->custom->call;
-		$expected = ['justOne' => false, 'w' => 1, 'wTimeoutMS' => 10000, 'fsync' => false];
-		$this->assertEqual('remove', $result['method']);
-		$this->assertEqual($expected, $result['params'][1]);
-
-		$this->_db = new MongoDb($this->_testConfig + ['w' => 1, 'wTimeoutMS' => 10000]);
-		$this->_db->server = new MockMongoConnection();
-		$this->_db->connection = new MockCallable();
-		$this->_db->connection->custom = new MockCallable();
-
-		$query = new Query(['type' => 'read', 'source' => 'custom']);
-		$this->_db->create($query);
-		$result = $this->_db->connection->custom->call;
-		$expected = [null, ['w' => 1, 'wTimeoutMS' => 10000, 'fsync' => false]];
-		$this->assertEqual('insert', $result['method']);
-		$this->assertEqual($expected, $result['params']);
-
-		$query = new Query(['type' => 'read', 'source' => 'custom', 'data' => ['something']]);
-		$this->_db->update($query);
-		$result = $this->_db->connection->custom->call;
-		$expected = [
-			'upsert' => false, 'multiple' => true, 'w' => 1, 'wTimeoutMS' => 10000, 'fsync' => false
-		];
-		$this->assertEqual('update', $result['method']);
-		$this->assertEqual($expected, $result['params'][2]);
-
-		$query = new Query(['type' => 'read', 'source' => 'custom']);
-		$this->_db->delete($query);
-		$result = $this->_db->connection->custom->call;
-		$expected = ['justOne' => false, 'w' => 1, 'wTimeoutMS' => 10000, 'fsync' => false];
-		$this->assertEqual('remove', $result['method']);
-		$this->assertEqual($expected, $result['params'][1]);
-	}
-
-	public function testGridFsCRUDWithDefaultPrefix() {
```

Comment:
This and the following tests are okay to be removed, as respondsTo is going away and we don't support GridFS anymore.